server/src/main/java/org/apache/druid/server/initialization/jetty/JettyServerModule.java

```diff
@@ -101,6 +103,7 @@
   private static final Logger log = new Logger(JettyServerModule.class);
 
   private static final AtomicInteger ACTIVE_CONNECTIONS = new AtomicInteger();
+  private static final AtomicReference<ThreadPool> JETTY_SERVER_THREAD_POOL = new AtomicReference<>(null);
```

Comment:
Why is `AtomicReference` used ? Thread pool of jetty is initialized during start up and won't be changed. I think we could declare this variable as `QueuedThreadPool` directly.

Reply:
Thanks! Addressed.